ed_sensor_integration/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.0.2)
 project(ed_sensor_integration)
 
-## Compile as C++11, supported in ROS Kinetic and newer
-add_compile_options(-std=c++11)
+## Compile as C++14, supported in ROS Melodic and newer
+add_compile_options(-std=c++14)
```

Comment:
Just curious. But isn't this the default in Melodic? And thus no need to set it explicitly?

Reply:
When I run ` g++ -dM -E -x c++  /dev/null | grep -F __cplusplus` on Ubuntu 18.04 it prints: `#define __cplusplus 201402L`. So I think indeed C++14 is the default in Ubuntu 18.04. @jlunenburg Could you run this on Ubuntu 16.04?

Reply:
The athome18 laptop with 16.04 shows `#define __cplusplus 199711L` so I guess it's indeed not necessary to set this

Reply:
We are removing it everywhere